dask_cuda/tests/test_ucx_options.py

```diff
@@ -50,6 +50,7 @@ def _test_global_option(seg_size):
             assert conf["SEG_SIZE"] == seg_size
 
 
+@pytest.mark.xfail(reason="https://github.com/rapidsai/dask-cuda/issues/627")
 def test_global_option():
     for seg_size in ["2K", "1M", "2M"]:
         p = mp.Process(target=_test_global_option, args=(seg_size,))
```

Comment:
Ok marked this test with `xfail` so we can get these changes out-the-door to fix other CI issues. Though raised issue ( https://github.com/rapidsai/dask-cuda/issues/627 ) to track/investigate

Reply:
I think this is ok.  @pentschev do you have any concerns?  we are still in burn down so if something does crop up we have time to handle

Reply:
For clarity cuDF's and RAFT's CI still fails without these changes, which is blocking a fair number of people. No objection to revisiting, but I do think we need to do this short term.

Reply:
This is weird, it never failed before, including in my nightly runs with both [UCX 1.9](https://github.com/pentschev/ucx-py-ci/issues/64) and [UCX master](https://github.com/pentschev/ucx-py-ci/issues/65). I'm actually concerned that this may be some error from a previous test that pytest is spilling over. I opened a PR to trigger CI in https://github.com/rapidsai/dask-cuda/pull/628 , let's hold on and see if that fails too before merging this.

Reply:
Agree it is weird. I don't recall seeing this error on this PR before that build either.

Ok though JFYI before this came up, Ben had used a merge command upthread ( https://github.com/rapidsai/dask-cuda/pull/623#issuecomment-847439931 ). So the bot may wind up merging this anyways if passes.

Reply:
Haha, yeah, it was too quick, so no worries. I'm just now wondering if this isn't a bit dangerous, one could do any changes to the PR after someone told the bot to merge, and unless somebody sees it in time, it would be merged.

Reply:
Yeah it probably should be associated with a commit hash

Reply:
Or just invalidated based on timestamp. If commit timestamp > gpucibot merge comment, invalidate automerge.

Reply:
I mean the time the commit was pushed, not the local commit timestamp.

Reply:
Submitted PR ( https://github.com/rapidsai/dask-cuda/pull/630 ) to revert this piece. Probably still fails, but at least that gives us something to work from